corehq/motech/static/motech/js/connection_settings_detail.js

```diff
@@ -129,7 +129,7 @@ hqDefine("motech/js/connection_settings_detail", [
                 .removeClass("d-none text-success")
                 .addClass("text-danger");
             $testResult.text(gettext(
-                'CommCare HQ was unable to make the request: '
+                'CommCare HQ was unable to make the request: ',
```

Comment:
Hi,

Didn't look at all changes but just noticed this. 
Isn't the command here incorrect and would include it in the translated message?

Reply:
it shouldn't as it's outside of the string

Reply:
Oh trailing comma to function parameters too. Wasn't expecting that.